toolkit/offchain/src/csl.rs

```diff
@@ -282,6 +323,15 @@ pub(crate) trait TransactionBuilderExt {
 		ex_units: ExUnits,
 	) -> Result<(), JsError>;
 
+	/// Adds minting of 1 token (with empty asset name) for the given script using reference script
+	fn add_mint_one_script_token_using_reference_script(
+		&mut self,
+		script_hash: &ScriptHash,
+		ref_input: &TransactionInput,
+		script_size: usize,
```

Comment:
script_hash, other param and script_size

I think it can be a parameter of type PlutusScript  instead and be just one param. If not, please group it because they really are related.

Reply:
yep, used the PlutusScript type here